pkg/cmd/api/api.go

```diff
@@ -75,7 +75,7 @@ func NewCmdApi(f *cmdutil.Factory, runF func(*ApiOptions) error) *cobra.Command
 
 			Placeholder values "{owner}", "{repo}", and "{branch}" in the endpoint
```

Comment:
```suggestion
			Placeholder values %[1]s{owner}%[1]s, %[1]s{repo}%[1]s, and %[1]s{branch}%[1]s in the endpoint
```

Reply:
One small point I can't leave a comment/suggestion on being outside of change boundaries might be switching `"graphql"` too as it is command invoked here.